SpatialGDK/Source/SpatialGDKServices/Public/LocalDeploymentManager.h

```diff
@@ -55,6 +55,7 @@ class FLocalDeploymentManager
 	IDirectoryWatcher::FDirectoryChanged WorkerConfigDirectoryChangedDelegate;
 
 private:
+	bool PreRunChecks();
```

Comment:
Is there some specific reason for PreRunChecks() to be a non-const member function? It seems it could be moved to anonymous namespace in LocalDeploymentManager.cpp? Or at least made static?

Reply:
I think making it static is fine.